java/client/src/main/java/glide/api/models/commands/SortBaseOptions.java

```diff
@@ -0,0 +1,112 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import java.util.ArrayList;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import lombok.experimental.SuperBuilder;
+
+/**
+ * Optional arguments to {@link glide.api.commands.GenericBaseCommands#sort(String, SortOptions)},
```

Comment:
Please add import and shorten the links
```suggestion
 * Optional arguments to {@link GenericBaseCommands#sort(String, SortOptions)},
```